app.py

```diff
@@ -1,37 +1,22 @@
 import os
 from logging import getLogger
 from fastapi import FastAPI, Response, status
-from vectorizer import Vectorizer, VectorInput
-from meta import Meta
+from transformer import Transformer, TextInput, TextsInput
```

Comment:
The aim of this PR is to simplify the modul by using a higher level library.  A lot of the functionality implemented by this module is already implemented by sentence transformers  (including robust configs)